bundles/org.eclipse.osgi/container/src/org/eclipse/osgi/internal/framework/EquinoxBundle.java

```diff
@@ -695,6 +695,16 @@ public ModuleClassLoader run() {
 		return new ResolvedModuleClassLoader(cl, report);
 	}
 
+	// This method is only preserved as a temporary workaround for consumers that
+	// reflectively access it until they are adapted, such as Jetty OSGi's
+	// DefaultBundleClassLoaderHelper
```

Comment:
Can we reference https://github.com/jetty/jetty.project/pull/14086 here maybe (or any other bug that tracks the problem at jetty)

Reply:
Of course. I added a reference to the Jetty PR with https://github.com/eclipse-equinox/equinox/pull/1211/commits/6a7fb39a0ce003c97e09734c595d9b5ae9c7f8d6

Awesome that you already made a proposal for a fix to Jetty! ❤️